go/tools/builders/link.go

```diff
@@ -161,3 +175,20 @@ func link(args []string) error {
 
 	return nil
 }
+
+var versionExp = regexp.MustCompile(`.*go1\.(\d+).*$`)
```

Comment:
Consider avoiding regex magic with stdlib: https://pkg.go.dev/go/version

Reply:
I decided to avoid this to keep the logic compilable on older Go versions than 1.22.

Reply:
makes sense -- disregard this comment. Thanks for the clarification 👍